tools/cldr-apps/js/src/index.js

```diff
@@ -6,14 +6,14 @@ import { createApp } from "vue";
 import AboutPanel from "./views/AboutPanel.vue";
 import WaitingPanel from "./views/WaitingPanel.vue";
```

Comment:
I think actually cldrAbout.js should import AboutPanel directly -- this detail is the concern of cldrAbout, not index.js

We can move the `views` folder inside `webapps` -- then just like the js files, the vue files will be pulled into the bundle automatically and will also be usable without bundling

Reply:
the views are part of the bundle.  I think we should drop cldrAbout.js eventually and have a new top level router for the single page app.

Reply:
cldrAbout might be a special case given how small it is

in general modules should be empowered to use vue without being tightly coupled with webpack

a new top level router sounds good -- I found it awkward to add the user id as in `v#recent_activity///1234`

Reply:
I think we need a phased approach and let's discuss this afternoon.

We're not rewriting all modules in Vue for 39, that seems clear. So let's take the current approach with the single top level entrypoint I'm using for those few items (About, Retry, and maybe Dashboard and Login) that use it, and then in 40 revisit a new top level router/top level SPA , ref CLDR-14398

Reply:
My comment about cldrAbout is really tangential to what this ticket does

I've tested this PR and it works for creating bundle.js that contains *everything* -- that's what we want, right? Is there any problem with this PR?

Reply:
Sorry i had forgotten the actual review !

Reply:
> cldrAbout might be a special case given how small it is
> 
> 
> 
> in general modules should be empowered to use vue without being tightly coupled with webpack
> 
> 
> 
> a new top level router sounds good -- I found it awkward to add the user id as in `v#recent_activity///1234`
> 
> 

Question about the bundler- have you tried using the browsers debugger against something in the bundle?

Reply:
Yes, check out the screenshot:
![image](https://user-images.githubusercontent.com/24991506/108898542-50e3d080-75e5-11eb-86e6-1fc9a2e0f0f3.png)